3rdparty/everest/README.md

```diff
@@ -0,0 +1 @@
+The files in this directory stem from [Project Everest](https://project-everest.github.io/) and are distributed under the Apache 2.0 license.
```

Comment:
Please add a few words to explain:

* What the code in this directory is for (X25519).
* Why one would want to use this code (formally verified, a lot faster than the default implementation).
* Why one would not want to use this code (only supported on [list of platforms], optimized for speed and not for code size).

Reply:
b6ead0227d0d1169162aee30036137df925abd20

Reply:
I added `armcc` to the list of supported platforms - is there an easy way for me to get a local copy of that, or perhaps a cloud VM that I can use temporarily?

Reply:
armcc doesn't compile for x86, only for arm CPUs. Until the code works with gcc and clang for some arm target, armcc support is a moot point.

Reply:
Done! 4ce6edb668e443457ca9073cded6bbbd8ba7e4bc